02-primitives/02-strings/exercise.yaml

```diff
@@ -0,0 +1,20 @@
+name: "strings"
+input:
+- cidr: "192.168.0.1/10"
```

Comment:
Are these actually valid CIDR prefix? I mean `192.168.0.1/10` should is "officially" written as `192.128.0.0/10`

Reply:
No, these are just random CIDR prefixes.